src/services/ISessionStorage.ts

```diff
@@ -75,7 +75,7 @@ export interface ISessionStorage {
    * @param user The AppUser value used to get the related fiscalCode
    */
   readonly getLollipopAssertionRefForUser: (
-    user: UserV5
+    user: User
```

Comment:
Because when we get the user from session we get a User, I changed the type only where necessary for my PR, but I think that it should be a User also on the other function.